macOSUpgrade.sh

```diff
@@ -423,4 +457,4 @@ else
 fi
 /bin/sleep 3
 
-cleanExit 0
+cleanExit 0
```

Comment:
no newline at end of line.

Reply:
There is no newline at the end of my branch. The end of file on both branches match. Is there something that I am missing?

Reply:
I have no idea for that.  it is not serious. Just for cosmetic issue on viewing web page.  Thank you for fixing it.